SchoolHousing/SchoolHousing/Pages/Index.cshtml.cs

```diff
@@ -10,17 +10,25 @@
 using ElementarySchoolResponse;
 public class IndexModel : PageModel
 {
-    public void OnGet()
+    public JsonResult OnGet()
     {
-        using (var webClient = new WebClient())
-        {
-            string schoolString = webClient.DownloadString("https://data.cityofchicago.org/resource/tj8h-mnuv.json");
+            string schoolString = getJsonString("https://data.cityofchicago.org/resource/tj8h-mnuv.json");
             var schoolData = ElementarySchool.FromJson(schoolString);
-            ViewData["ElementarySchool"] = schoolData;
+          
 
-            string houseString = webClient.DownloadString("https://data.cityofchicago.org/resource/s6ha-ppgi.json");
+            string houseString = getJsonString("https://data.cityofchicago.org/resource/s6ha-ppgi.json");
             var houseData = AffordableHouse.FromJson(houseString);
-            ViewData["AffordableHouse"] = houseData;
+           
+
+        return new JsonResult(schoolData);
+
+    }
+
+    private string getJsonString(String url)
```

Comment:
Extracting Get json logic as a method

Reply:
In .net, method names should be PascalCase.